packages/wdio-mocha-framework/src/constants.js

```diff
@@ -20,3 +20,8 @@ export const EVENTS = {
 }
 
 export const NOOP = /* istanbul ignore next */ function () {}
+export const MOCHA_TIMEOUT_MESSAGE = 'For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.'
+export const MOCHA_TIMEOUT_MESSAGE_REPLACEMENT = [
+    'The execution in the test "%s %s" took too long. Try to reduce the run time or',
+    'increase your timeout for test specs (https://webdriver.io/docs/timeouts.html).'
```

Comment:
can't this be a single string?

Reply:
it is a single string (see `join(' ')`) - I just write it this way to allow having it in multiple lines. You could theoretically also do a string concatenation but I prefer this.